sales.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <meta charset="UTF-8">
+  <title>Sales</title>
+  <nav>
+    <ul id="navigation">
+      <li><a href='file:///Users/codyhulsey/projects/cookie_stand_201/mainpage.html'>HOME</a></li>
+      <li><a href="file:///Users/codyhulsey/projects/cookie_stand_201/index.html">SALES</a></li>
+      <li>
+      <a href="file:///Users/codyhulsey/projects/cookie_stand_201/styleguide.html">STYLE GUIDE</a></li>
+    </ul>
+  </nav>
+</head>
+<body>
+  <form id="myForm">
+    <fieldset>
```

Comment:
Good exercise using the `<fieldset>` elements, but might be a bit much to segregate all of these inputs into their own group, given that all of them together are used to make one store object.